src/classes/user.py

```diff
@@ -39,7 +39,8 @@ def parse_rootme_user_data(data):
         idx = int(data["id_auteur"])
         username = data["nom"]
         score = int(data["score"])
-        rank = int(data["position"])
+        _rank = data["position"]
+        rank = int(_rank) if _rank != "" else 9999999
```

Comment:
Just a quick question : how did you chose this default value ?

Reply:
it is absolutely arbitrary, 
I could have used -1 for more explicit "Unknown position" but it would have mess with a ranking: placing -1 people above those with != 0 score
Maybe I could use None but we'd also have to deal with the case ourselves

Also I plan to change it to a far better value when implementing a real DB

Reply:
Ok thanks for the precision !